.github/workflows/UnitTests.yml

```diff
@@ -11,22 +11,20 @@ jobs:
     strategy:
       matrix:
         configs:
-          # Linux clang 15 runners are temporarily disabled due to installation issues. See https://github.com/sonible/VCTR/issues/86
-          # - runner: ubuntu-22.04
-          #  cc: clang-15
-          #  cxx: clang++-15
-          #  ipp: no_ipp
+          - runner: ubuntu-22.04
+            cc: clang-15
+            cxx: clang++-15
+            ipp: no_ipp
 
           - runner: ubuntu-24.04
             cc: gcc-13
             cxx: g++-13
             ipp: no_ipp
 
-          # Linux clang 15 runners are temporarily disabled due to installation issues. See https://github.com/sonible/VCTR/issues/86
-          # - runner: ubuntu-22.04
-          #  cc: clang-15
-          #  cxx: clang++-15
-          #  ipp: ipp
```

Comment:
Found out that the clang installation issues have been fixed in the meantime as clang-15 is now pre-installed on the ubuntu runners, so finally solved that old issue along the way